bylaws/007-psr-naming-conventions.md

```diff
@@ -5,17 +5,18 @@ Naming conventions for code released by PHP FIG
 2. Abstract classes MUST be prefixed by `Abstract`: e.g. `Psr\Foo\AbstractBar`.
 3. Traits MUST be suffixed by `Trait`: e.g. `Psr\Foo\BarTrait`.
 4. PSR-1, 4, and 12 MUST be followed.
-5. The vendor namespace MUST be `Psr`.
-6. There MUST be a package/second-level namespace in relation with the PSR that
+5. The vendor namespace for code released as part of a PSR MUST be `Psr`.
+6. The vendor namespace for code released as part of a PER MUST be `Fig`.
```

Comment:
Why the discrepancy here? If a PSR namespace is `Psr`, why wouldn't a PER namespace be `Per`?

Reply:
Mainly because I was using PERs as a place for util libraries to live, and that already has an established `Fig` namespace.

If we split those up into 3 separate tracks, then I don't know off hand if it makes sense for PERs to have their own namespace or not.  Many (eg, coding standards) likely won't even have code artifacts.